_posts/2015-03-16-CocoaPods.org-Take-Two.md

```diff
@@ -0,0 +1,151 @@
+
+```json
+// http://metrics.cocoapods.org/api/v1/pods/MiniFuture
+
+{
+	"cocoadocs": {
+	    "created_at": "2015-03-08 16:48:25 UTC",
+	    "doc_percent": 22,
+	    "dominant_language": "Swift",
+	    "download_size": 416,
+	    "initial_commit_date": "2015-03-01 12:03:09 UTC",
+	    "license_canonical_url": "http://opensource.org/licenses/MIT",
+	    "license_short_name": "MIT",
+	    "notability": 0,
+	    "quality_estimate": 50,
+	    "readme_complexity": 72,
+	    "rendered_readme_url": "http://cocoadocs.org/docsets/MiniFuture/0.1.0/README.html",
+	    "total_comments": 8,
+	    "total_files": 4,
+	    "total_lines_of_code": 327,
+	    "total_test_expectations": 204,
+	    "updated_at": "2015-03-08 16:48:25 UTC"
+	},
+	"github": {
+	    "contributors": 1,
+	    "created_at": "2015-03-01 10:09:08 UTC",
+	    "forks": 0,
+	    "language": "Swift",
+	    "open_issues": 0,
+	    "open_pull_requests": 0,
+	    "stargazers": 8,
+	    "subscribers": 2,
+	    "updated_at": "2015-03-05 01:01:44 UTC"
+	}
+}
+```
+* A preview image is generated for the library for Social Media, chat, or really anything that supports open graph previews.
+  {% breaking_image http://cocoadocs.org/docsets/MiniFuture/0.1.0/preview.png %}
+  
+* We then generate an estimated quality number for the library. This is based on a collection of [individual metrics][7] that are applied to the generated metrics data. They take a library and add or remove points based whether the rule applies for that library. These range from the popularity of the library, total test expectations / lines of code to the files, and average lines of code per file in the library.
+
+```json
+// https://cocoadocs-api-cocoapods-org.herokuapp.com/pods/MiniFuture/stats
+
+[
+	{
+	    "applies_for_pod": false,
+	    "description": "Testing a library shows that the developers care about long term quality on a project as internalized logic is made explicit via testing.",
+	    "modifier": -20,
+	    "title": "Test Expectations / Line of Code"
+	},
+	{
+	    "applies_for_pod": false,
+	    "description": "Too big of a library can impact startup time, and add redundant assets.",
+	    "modifier": -10,
+	    "title": "Download size"
+	},
+	{
+	    "applies_for_pod": false,
+	    "description": "Smaller, more composeable classes tend to be easier to understand.",
+	    "modifier": -8,
+	    "title": "Lines of Code / File"
+	},
+	[...]
+	{
+	    "applies_for_pod": true,
+	    "description": "A popular library means there can be a community to help improve and maintain a project.",
+	    "modifier": 5,
+	    "title": "Is popular"
+	}
+]
+```
+
+This gives enough metadata to start creating a page that represents the library with the kind of context that is specific to our community. In a lot of ways it can be better than the front page of the GitHub repo because of this.
```

Comment:
"because of this" kind of awkward/unnecessary. I'd drop it or move it to the beginning of the sentence.